packages/gafl-webapp-service/src/pages/order-complete/licence-details/__tests__/licence-details.spec.js

```diff
@@ -0,0 +1,70 @@
+import { getData } from '../route.js'
+import Boom from '@hapi/boom'
+import { COMPLETION_STATUS } from '../../../../constants.js'
+import { CONCESSION, CONCESSION_PROOF, LICENCE_TYPE } from '../../../../processors/mapping-constants.js'
+
+const createMockRequest = status => ({
+  cache: jest.fn(() => ({
+    helpers: {
+      status: {
+        get: jest.fn(() => status)
+      },
+      transaction: {
+        getCurrentPermission: jest.fn(() => ({
+          startDate: '2020-06-06',
+          endDate: '2021-06-05',
+          licenceType: LICENCE_TYPE['trout-and-coarse'],
+          numberOfRods: '3',
+          concessions: [{
+            type: CONCESSION.DISABLED,
+            proof: {
+              type: CONCESSION_PROOF.blueBadge,
+              referenceNumber: '123456324'
+            }
+          }, {
+            type: CONCESSION.SENIOR,
+            proof: {
+              type: CONCESSION_PROOF.none
+            }
+          }]
+        }))
+      }
+    }
+  }))
+})
```

Comment:
Please could you move this to the end of the file? It's purely stylistic, but then you're straight into the tests and can look for this should you need to